frontend/src/metabase/components/Popover.jsx

```diff
@@ -128,7 +128,7 @@ export default class Popover extends Component {
 
   _popoverComponent() {
     const childProps = {
-      maxHeight: this._getMaxHeight(),
+      "max-height": this._getMaxHeight(),
```

Comment:
⚠️ Please check visually on review.

Was getting a console warning about this.

Checked height of Popover, as far as could tell it looks like it did before.